_ml-commons-plugin/algorithms.md

```diff
@@ -0,0 +1,220 @@
+* Fixed in time (FIT) RCF: Detects anomalies in time series data.
+
+### Parameters
+
+#### Batch RCF
+
+Parameter | Type   | Description | Default Value
+:--- |:--- | :--- | :---
+number_of_trees | integer | Number of trees in the forest | 30
+sample_size | integer | The same size used by the stream samplers in the forest | 256
+output_after | integer | The number of points required by stream samplers before results return | 32
+training_data_size | integer | The size of your training data | Data set size
+anamoly_score_threshold | double | The threshold of the anomaly score | 1.0 
+
+#### Fit RCF
+
+Parameter | Type   | Description | Default Value
+:--- |:--- | :--- | :---
+number_of_trees | integer | Number of trees in the forest | 30
+shingle_size | integer | A shingle, or consecutive sequence of the most recent records | 8
+sample_size | integer | The sample size used by stream samplers in the forest | 256
+output_after | integer | The number of points required by stream samplers before results return | 32
+time_decay | double | The decay factor used by stream samplers in the forest | 0.0001 
+anomaly_rate | double | The anomaly rate | 0.005
+time_field | string | (**Required**) The time filed for RCF to use as time-series data | N/A
+date_format | string | The date and time format for the time_field field | "yyyy-MM-ddHH:mm:ss"
+time_zone | string | The time zone for the time_field field | "UTC" 
+
+
+### APIs
+
+* [Train]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/#train-model)
+* [Predict]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/#predict)
+* [Train and predict]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/#train-and-predict)
+
+
+### Limitations
+
+For FIT RCF, you can train the model with historical data, and store the trained model in your index. The model will be deserialized and predict new data points when using the Predict API. However, the model in the index will not be refreshed with new data, because the model is "Fixed In Time".
+
+## Localization 
+
+The Localization algorithm finds subset level information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes or anomalies. Localization can be applied in different scenarios, such as data exploration or root cause analysis, to expose the contributors driving the activity of interest in the aggregate data.
+
+### Parameters
+
+Parameter | Type   | Description | Default Value
+:--- | :--- | :--- | :---
+index_name | String | The data collection to analyze | N/A
+attribute_field_names | List<String> | The fields for entity kets | N/A
+aggregations | List<AggregationBuilder> | The fields and aggregation for values | N/A
+time_field_name | String | The timestamp field | null
+start_time | Long | The beginning of the time range | 0 
+end_time | Long | The end of time range | 0
+min_time_interval | Long | The minimal time interval/scale for analysis | 0
```

Comment:
Should it be "minimum" instead of "minimal"? Another @natebower question.

Reply:
Yes, it should be "minimum".